iac/terraform/aws/auth/lambdas.tf

```diff
@@ -8,6 +8,15 @@ locals {
       env_variables = {
         foo = "bar"
       }
+    },
+    register-org = {
+      name     = "register-org"
+      zip_path = "${path.module}/../../../../core/services/aws/.build/zips/registerOrg.zip"
+      policy   = data.aws_iam_policy_document.lambda_common_policy
+      handler  = "registerOrg.default"
+      env_variables = {
+        foo = "bar"
```

Comment:
Why do we need env var placeholder?

Reply:
this is list of environment variables for lambda. this was example variable. removed it